example/config.cpp

```diff
@@ -281,6 +281,8 @@ static void parse_options(config_data *data, const kora::config_t &options) {
 	data->cfg_state.removal_delay = options.at("removal_delay", 0);
 	data->cfg_state.server_prio = options.at("server_net_prio", 0);
 	data->cfg_state.client_prio = options.at("client_net_prio", 0);
+	data->cfg_state.grpc_address = options.at<const char *>("grpc_address");
```

Comment:
What guarantees that this pointer will be correct and available?

Reply:
123ea1f